DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Dashboard/Index.cshtml

```diff
@@ -17,214 +18,222 @@
-            <dt class="nhsuk-summary-list__key">
-              Centre support details
-            </dt>
-            <dd class="nhsuk-summary-list__value">
-              @Model.CentreDetails.BannerText
-            </dd>
-          </div>
-
-          <div class="nhsuk-summary-list__row details-list-with-button__row">
-            <dt class="nhsuk-summary-list__key">
-              Your IP address
-            </dt>
-            <dd class="nhsuk-summary-list__value">
-              @Model.CentreDetails.IpAddress
-            </dd>
-          </div>
-
-          <div class="nhsuk-summary-list__row details-list-with-button__row">
-            <dt class="nhsuk-summary-list__key">
-              Pre-approved IPs
-            </dt>
-            <dd class="nhsuk-summary-list__value">
-              @Model.CentreDetails.ApprovedIps
-            </dd>
-          </div>
-
-        </dl>
-
-        <a class="nhsuk-button" role="button">
-          Edit centre details
-        </a>
-      </div>
-    </details>
-
-    <ul class="nhsuk-grid-row nhsuk-card-group">
-      <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
-
-        <div class="nhsuk-card nhsuk-card--clickable">
-          <div class="nhsuk-card__content">
-            <h2 class="nhsuk-card__heading nhsuk-heading-m">
-              <a class="nhsuk-card__link" href="#">Contract details</a>
-            </h2>
-            <p class="nhsuk-card__description">See details about your contract type</p>
-          </div>
-        </div>
-
-      </li>
-      <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
-
-        <div class="nhsuk-card nhsuk-card--clickable">
-          <div class="nhsuk-card__content">
-            <h2 class="nhsuk-card__heading nhsuk-heading-m">
-              <a class="nhsuk-card__link" href="#">Reports</a>
-            </h2>
-            <p class="nhsuk-card__description">See reports about your usage and evaluations</p>
-          </div>
-        </div>
-
-      </li>
-      <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
-
-        <div class="nhsuk-card nhsuk-card--clickable">
-          <div class="nhsuk-card__content">
-            <h2 class="nhsuk-card__heading nhsuk-heading-m">
-              <a class="nhsuk-card__link" href="#">Top courses</a>
-            </h2>
-            <p class="nhsuk-card__description">See information about your top courses</p>
-          </div>
-        </div>
-
-      </li>
-      <li class="nhsuk-grid-column-one-half nhsuk-card-group__item">
-
-        <div class="nhsuk-card nhsuk-card--clickable">
-          <div class="nhsuk-card__content">
-            <h2 class="nhsuk-card__heading nhsuk-heading-m">
-              <a class="nhsuk-card__link" href="#">Centre ranking</a>
-            </h2>
-            <p class="nhsuk-card__description">See centre rankings by week, month, or year</p>
-          </div>
-        </div>
-
-      </li>
-    </ul>
+<div class="nhsuk-grid-column-one-quarter">

```

Comment:
Lots of line ending and indentation changes on this file. Only new addition is the changes to this one-quarter div, and another highlighted change near the bottom of the page.

Reply:
Something has broken here to cause the incorrect indentation. Will mark a set of subsequent comments of issues I see.